collect_app/src/main/java/org/odk/collect/android/activities/AndroidShortcutsActivity.java

```diff
@@ -14,44 +14,44 @@
 
 package org.odk.collect.android.activities;
 
-import androidx.appcompat.app.AlertDialog;
-import android.content.DialogInterface;
-import android.content.DialogInterface.OnCancelListener;
 import android.content.Intent;
-import android.database.Cursor;
 import android.net.Uri;
 import android.os.Bundle;
 import android.os.Parcelable;
+
+import androidx.appcompat.app.AlertDialog;
 import androidx.appcompat.app.AppCompatActivity;
 
 import org.odk.collect.android.R;
-import org.odk.collect.android.dao.FormsDao;
+import org.odk.collect.android.forms.Form;
+import org.odk.collect.android.forms.FormsRepository;
+import org.odk.collect.android.injection.DaggerUtils;
 import org.odk.collect.android.provider.FormsProviderAPI.FormsColumns;
 
 import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
 
 /**
- * Allows the user to create desktop shortcuts to any form currently avaiable to Collect
+ * Allows the user to create desktop shortcuts to any form currently available to Collect
  *
  * @author ctsims
  * @author carlhartung (modified for ODK)
  */
-public class AndroidShortcuts extends AppCompatActivity {
+public class AndroidShortcutsActivity extends AppCompatActivity {
+
+    @Inject
+    FormsRepository formsRepository;
 
     private Uri[] commands;
     private String[] names;
 
     @Override
     public void onCreate(Bundle bundle) {
         super.onCreate(bundle);
-
-        final Intent intent = getIntent();
-        final String action = intent.getAction();
-
-        // The Android needs to know what shortcuts are available, generate the list
-        if (Intent.ACTION_CREATE_SHORTCUT.equals(action)) {
```

Comment:
I'm not sure why this filter was there seeing as that's the only reason the Activity would be opened.